GPU/Common/PostShader.cpp

```diff
@@ -235,6 +230,12 @@ void LoadPostShaderInfo(Draw::DrawContext *draw, const std::vector<Path> &direct
 	}
 	shaderInfo.insert(shaderInfo.begin(), off);
 
+	textureShaderInfo.clear();
+	TextureShaderInfo textureOff{};
+	textureOff.name = "Off";
+	textureOff.section = "Off";
+	textureShaderInfo.push_back(textureOff);
```

Comment:
Hm, should this be insert too?

-[Unknown]

Reply:
Uh yeah, duh.